fenix/app/src/main/res/layout/tab_tray_grid_item.xml

```diff
@@ -102,7 +102,6 @@ A FrameLayout here is an efficient way of having a views stack while allowing:
                     android:layout_height="match_parent"
                     android:importantForAccessibility="no"
                     android:padding="22dp"
-                    app:srcCompat="@drawable/mozac_ic_globe"
```

Comment:
should the entire `ImageView` just be removed?

The icon in the sister list_item XML will need to be cleaned-up, too.

Reply:
We need the `TabThumbnailView` still to attach the thumbnail but I can get rid of the unnecessary `ImageView` with the id `default_tab_thumbnail`
on both the grid item and `tab_tray_item`

Reply:
Shouldn't we ask UX before we remove the globe placeholder? I don't see any confirmation about this on the bug besides comment 6 from verdi, and this was before we learnt that the cause was Compose inside a RecyclerView.

If there was confirmation, please link it in the PR and add remember to add the label for UX to review before we merge.

Reply:
I'll separate out the globe image removal to a separate PR so we can go ahead with this upgrade (and the following PR to apply the upgrades to the other RecyclerViews in the app).